### DISCUSSION TITLE
QuarkChain L2 testnet 是以后的主力 testnet 么？

### DISCUSSION THREAD

**Discussion by linghuccc** [IntensiveCoLearning/Web3-URL#212] [Q&A] [answered]
因为听说 W3Q Galileo 会停用，所以我试着用 QuarkChain L2 testnet，但是 Chainlist 里面的 QuarkChain L2 testnet 似乎缺少 RPC 的信息：
![quark2](https://github.com/user-attachments/assets/08167c0c-e91a-43c8-995e-a8e5526007c6)

我的 MetaMask 里面的 QKC 也总是显示有问题：
![quark1](https://github.com/user-attachments/assets/844a5ffe-50a5-4ca4-baf1-ff4c7e820ac0)

**Comment by qzhodl** [comment]
chainlist里面的比较老了，可以参考这个文档：https://github.com/ethstorage/pm/blob/main/L2/testnet_usage.md

**Reply by linghuccc** [reply]
是的，我就是按照这个文档配置的 MetaMask，但是正如图 2 显示，我的 QKC 总是显示红色三角。

确认一下，QuarkChain L2 testnet 的 链 ID 是 43069 吧？

另外想问一下，在 ethfs-cli 文档的 Supported networks （https://github.com/ethstorage/ethfs-cli/）里面，是没有 QuarkChain L2 testnet 的，但是在 L2 testnet usage 文档（https://github.com/ethstorage/pm/blob/main/L2/testnet_usage.md）里面，有下面的 command：
// upload your application using file upload type 2
$ ethfs-cli upload -f <your application folder> -a <flat directory address> -c 43069 -p <private key> -t 2

这是不是说明 ethfs-cli 其实是支持 QuarkChain L2 testnet，只是文档里面漏写了？

**Reply by qzhodl** [reply, marked as answer]
1. Chain ID 是 43069
2. qkc 显示红色三角是因为symbol有冲突的警告，不影响使用
3. ethfs-cli 已经支持，文档稍后会更新

**Reply by linghuccc** [reply]
> 1. Chain ID 是 43069
> 2. qkc 显示红色三角是因为symbol有冲突的警告，不影响使用
> 3. ethfs-cli 已经支持，文档稍后会更新

明白，因为我昨天 transfer QKC 失败，所以担心是不是 network configure 错了。

最后问一个问题，Custom Gas Token 0x4200000000000000000000000000000000000800 （WSC）是干什么的，QuarkChain L2 testnet 的 gas token 不是 QKC 么？

**Reply by qzhodl** [reply]
是 quarkchain l2 一个新的feature, 可以理解为不能transfer的gas token，详情可以看这个：https://perfect-amphibian-929.notion.site/Soul-ETH-A-Soulbound-Gas-Token-for-Rollups-acbc11492bc145849c470a8ce5114128
